<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}

QLabel, QTextEdit {
    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.1);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dragDropArea">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>400</width>
      <height>150</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#dragDropArea {
    background-color: #f0f0f0;
    border: 2px dashed #aaa;
    border-radius: 10px;
    color: #555;
    font-size: 18px;
    text-align: center;
    padding: 10px;
}
QLabel#dragDropArea:hover {
    background-color: #e0e0e0;
    border-color: #888;
}
</string>
    </property>
    <property name="text">
     <string>拖入PCD文件</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultTextEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>400</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit#resultTextEdit {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 10px;
    font-size: 16px;
    color: #333;
}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#saveButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    padding: 10px 24px;
    text-align: center;
    font-size: 16px;
    border-radius: 5px;
}

QPushButton#saveButton:hover {
    background-color: #45a049;
}
</string>
    </property>
    <property name="text">
     <string>保存结果为TXT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
